test/CNVCallingAndMergeForFabric/test_base_outputs.json

```diff
@@ -0,0 +1,16 @@
+{
+  "CNVCallingAndMergeForFabric.filtered_cnv_genotyped_segments_vcf": {
+    "file": "gs://palantir-workflows-test-data/CNVCallingAndMergeForFabric/0437227296_subset.filtered.genotyped-segments.vcf.gz",
+    "line_skip_regex": "^##bcftools|^##GATKCommandLine="
+  },
+  "CNVCallingAndMergeForFabric.filtered_cnv_genotyped_segments_vcf_index": null,
+  "CNVCallingAndMergeForFabric.filtered_cnv_genotyped_segments_vcf_md5sum": null,
+  "CNVCallingAndMergeForFabric.merged_vcf": {
+    "file": "gs://palantir-workflows-test-data/CNVCallingAndMergeForFabric/0437227296_subset.merged.vcf.gz",
+    "line_skip_regex": "^##bcftools|^##GATKCommandLine=|^##FORMAT=<ID=GT|^##source="
```

Comment:
Why skip the `FORMAT=<ID=GT` line in the header? Is this not standardized so it fails because there's a different description or something?